sys/net/crosslayer/netopt/netopt.c

```diff
@@ -93,6 +93,20 @@ static const char *_netopt_strmap[] = {
     [NETOPT_BLE_CTX]               = "NETOPT_BLE_CTX",
     [NETOPT_CHECKSUM]              = "NETOPT_CHECKSUM",
     [NETOPT_PHY_BUSY]              = "NETOPT_PHY_BUSY",
+    [NETOPT_LORAWAN_APPEUI]        = "NETOPT_APPEUI",
```

Comment:
there are some broken references here (e.g NETOPT_APPEUI instead of NETOPT_LORAWAN_APPEUI)

Reply:
fixed and directly amended.